### ISSUE TITLE
Synopsis example doesn't work on networked boxes

### ISSUE THREAD

**Issue by zoffixznet** [tokuhirom/p6-HTTP-Server-Tiny#41]
Had some issues running the synopsis code: server was accessible from localhost but not from other boxes on the network. It should bind to '0.0.0.0', not '127.0.0.1'

**Comment by jonathanstowe** [comment]
That's right.  I'm not sure it's even an issue to be honest as the ```host``` is the address it listens on so that is expected.

**Comment by zoffixznet** [comment]
>  I'm not sure it's even an issue

Well, if a stranger on IRC didn't point this out to me, I would've assumed this module to be broken and moved on to something else.

**Comment by jonathanstowe** [comment]
I'll put a comment in the synopsis, you can bet your arse that if we put 0.0.0.0 then some paranoid type will start whining about security.
